src/org-manager/containers/home/organisations-home.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="hmcts-width-container">
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <h1 class="hmcts-page-heading__title govuk-heading-xl">
+      Organisation approvals
+    </h1>
+
+    <!-- The global search component -->
+    <!-- <app-search-organisations-form
+      [searchString]="searchString" (submitForm)="submitSearch($event)">
+    </app-search-organisations-form> -->
```

Comment:
why are we committing commented out code? Needs a good reason IMO

Reply:
It's going to be needed later but I already removed the code in the redundant code in the .ts file so I'll remove this too. Good call.